assets/src/adaptivity/operators/contains.ts

```diff
@@ -5,6 +5,14 @@ export const parseNumString = (item: string): string | number => {
   return !Number.isNaN(parseFloat(item)) ? parseFloat(item) : item.toString().trim();
 };
 
+export const parseArrayString = (value: (string | number)[] | string): (string | number)[] => {
```

Comment:
in utils/common I think we already have a stringy array parser? we can probably move `parseNumString` into there as well. also lets start getting into the habit of writing unit tests for these as we add them